app/src/components/LayerControlBody.vue

```diff
@@ -610,7 +710,43 @@ export default {
         name: `flip on axis ${axis}`
       })
       this.flipAxis({ axis })
+    }, 
+    setResolution: function () {
+      const isotropic = +this.selectedResolutionX === +this.selectedResolutionY && +this.selectedResolutionY === +this.selectedResolutionZ
```

Comment:
I am somewhat confused. 

isotropic is a state that's set by the user, not computed on the fly. 

This should be a simple check `if (this.isotropic)` ? Or am I missing something?